### PR TITLE
upgrade to cypress 13.11.0

### PR DESCRIPTION
**Pull request by pjhill**
## Summary
Upgrading Cypress from 12.17.4 to 13.11.0. In order to get Cypress upgraded we needed to update the install action to use the new v13 cypress binary in our private S3 bucket -- `vetsgov-website-builds-s3-upload.s3.us-gov-west-1.amazonaws.com`

Please note that many keyboard navigation tests needed to be marked skipped for now as the keyboard navigation functionality appears to be experiencing issues that cause tests to hang. This issue is difficult to troubleshoot as it appears to occur erratically in local testing. In order to not hold up the upgraded Cypress version being integrated into `vets-website` for the other 700 test specs, these dozen or so specs have been skipped for now.

### Skipped Tests Resolution
In order to define a path toward re-enabling the skipped tests, I have created the below tickets to track the work and request fixes from the teams that own these test specs --
- [MHV Secure Messaging Team](https://app.zenhub.com/workspaces/mhv-to-vagov-secure-messaging-mhv-team-625d8e7aa8f28c002003aefe/issues/gh/department-of-veterans-affairs/va.gov-team/86214)
- [Benefits Education Team](https://github.com/department-of-veterans-affairs/va.gov-team/issues/86227)
- 

## Related issue(s)
[Upgrade Cypress to 13.x in `vets-website`](https://github.com/department-of-veterans-affairs/va.gov-team/issues/72406)

## Testing done
Tested with PR CI runs by specifically referring to `upgrade_cypress_13` branch during checkout to get access to the updated workflows/install action. This will be removed before merging to `main`.

## Screenshots
None

## What areas of the site does it impact?
None

### Quality Assurance & Testing
No impact to Veteran facing site.

### Error Handling
N/A

### Authentication
N/A

## Requested Feedback
None

### CODE REVIEW

File: src/applications/edu-benefits/feedback-tool/tests/00.feedback-tool.cypress.spec.js [hunk 1]
```diff
@@ -91,8 +92,8 @@ describe('Feedback Tool Test', () => {
 
     cy.get(
       'input[name="root_educationDetails_school_view:manualSchoolEntry_name"]',
-      { timeout: Timeouts.slow },
-    );
+      { timout: Timeouts.slow },
```
Comment (adamwhitlock1) [inline src/applications/edu-benefits/feedback-tool/tests/00.feedback-tool.cypress.spec.js:95]: missing the `e` in timeout here

Comment (pjhill) [reply src/applications/edu-benefits/feedback-tool/tests/00.feedback-tool.cypress.spec.js]: Thanks!

File: src/applications/mhv-secure-messaging/tests/e2e/keyboard-nav-test/secure-messaging-compose-error-message-keyboard-nav.cypress.spec.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import PatientComposePage from '../pages/PatientComposePage';
 import FolderLoadPage from '../pages/FolderLoadPage';
 import { AXE_CONTEXT, Data } from '../utils/constants';
 
-describe('Secure Messaging Compose Errors Keyboard Nav', () => {
+describe.skip('Secure Messaging Compose Errors Keyboard Nav', () => {
```
Comment (fazilqa) [inline src/applications/mhv-secure-messaging/tests/e2e/keyboard-nav-test/secure-messaging-compose-error-message-keyboard-nav.cypress.spec.js:7]: as discussed [here](https://dsva.slack.com/archives/C03ECSBGSKX/p1718720331175189) all skipped tests will be fixed after merge

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Pull request merged**